tests/OAuth/ResourceOwner/AppleResourceOwnerTest.php

```diff
@@ -59,6 +59,54 @@ public function testHandleRequest(): void
         $this->assertTrue($resourceOwner->handles($request));
     }
 
+    /**
+     * @dataProvider provideAccessTokenData
+     */
+    public function testGetAccessTokenWithAutoGeneratedClientSecret(string $response, string $contentType): void
+    {
+        $resourceOwner = $this->createResourceOwner(
+            [
+                'client_secret' => 'auto',
+                'auth_key' => <<<key
+-----BEGIN PRIVATE KEY-----
+MHcCAQEEIEF2z9dMGTENgJmKXp8RXvAjQxk0/S4vXIRmUSn4nrCioAoGCCqGSM49
+AwEHoUQDQgAEqfbSt6dZgD5tT46T81FxvLa9d9k7tXiU7W6gee6GGCg9Q1xACLxb
+hphWMrna87Y1gYYyinFUJII1qjZ7fE6zzQ==
+-----END PRIVATE KEY-----
+key,
```

Comment:
A valid key is required by `JWT` so I generated a random one with https://8gwifi.org/jwsgen.jsp